standalone/include/psen_scan_v2_standalone/udp_client.h

```diff
@@ -33,7 +33,6 @@
 #endif
 
 #include <boost/asio.hpp>
-#include <boost/asio/high_resolution_timer.hpp>
 #include <boost/bind.hpp>
```

Comment:
```suggestion
```

Maybe also replace `boost::bind` by `std::bind`? (See https://stackoverflow.com/questions/8924149/should-stdbind-be-compatible-with-boostasio)

Reply:
Surely could be done. But I try to keep it simple by starting in small steps and removing the obvious ones first.